awx/ui_next/src/components/CodeEditor/CodeEditor.jsx

```diff
@@ -11,6 +13,8 @@ import { t } from '@lingui/macro';
 import styled from 'styled-components';
 import debounce from '../../util/debounce';
 
+config.set('loadWorkerFromBlob', false);
```

Comment:
This seems to address the CSP error. My only concern is unknowingly breaking some functionality w/ the code editors somewhere (although I think we're _probably_ fine).

cc @keithjgrant